File: Modules/CodeCamp/Components/RouteMapper.cs

using DotNetNuke.Web.Api;

namespace WillStrohl.Modules.CodeCamp.Components
{
    public class RouteMapper : IServiceRouteMapper
    {
        public void RegisterRoutes(IMapRoute mapRouteManager)
        {
            mapRouteManager.MapHttpRoute("CodeCamp", "default", "{controller}/{action}", new[] { "WillStrohl.Modules.CodeCamp.Services" });
        }
    }
}

File: Modules/WillStrohl.LightboxGallery/RouteMapper.vb

Imports DotNetNuke.Web.Api

Namespace WillStrohl.Modules.Lightbox

    Public Class RouteMapper
        Implements IServiceRouteMapper

        Public Sub RegisterRoutes(ByVal mapRouteManager As IMapRoute) Implements IServiceRouteMapper.RegisterRoutes
            mapRouteManager.MapHttpRoute("WillStrohl.LightboxGallery", "default", "{controller}/{action}", {"WillStrohl.Modules.Lightbox.Services"})
        End Sub

    End Class

End Namespace